locale/en/LC_MESSAGES/django.po

```diff
@@ -3280,14 +3280,44 @@ msgstr ""
 msgid "Yes, only in Tournai (10 EUR)"
 msgstr ""
 
-msgid ""
-"You already have a doctorate admission in progress, please delete it before "
-"creating a newer one, or contact your domain doctoral committee."
+msgid "You cannot fill in a language more than once, please correct the form."
 msgstr ""
 
-msgid "You cannot fill in a language more than once, please correct the form."
+#, python-format
+msgid ""
+"You cannot have more than %(max)s applications in progress at the same time."
 msgstr ""
 
+#, python-format
+msgid ""
+"You cannot submitted this admission for a continuing education as you "
```

Comment:
You cannot submit

Reply:
Je l'ai fait à chaque fois en plus, je modifie ça.